packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -458,6 +481,21 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
               }, 0)`}
               className={this._classNames.yAxis}
             />
+            <SVGTooltipText
```

Comment:
How much is the x and y axis shifting if both x and y axis are present.

Reply:
for yaxis title = 8px from left + 8px from right + to render the title
for xaxis title = 8px from bottom + 8px from top + to render the title